src/components/AddTask/AddTask.jsx

```diff
@@ -1,13 +1,33 @@
-import React from "react";
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
 import styles from "./style.module.scss";
 
-function AddTask() {
+const AddTask = () => {
+  const [text, setText] = useState("");
```

Comment:
do we really need this state in component in this case?

Reply:
Why not? 
We'll need to keep track of the data the user is typing into the "Add Todo" input box.